### ISSUE TITLE
Optimize README images

### ISSUE THREAD

**Issue by takahirom** [DroidKaigi/conference-app-2022#261]
**Idea Description**
Resize and use some tool to optimize it 🙏 

**Reference images and links**

**Comment by Nthily** [comment]
How about this [imgbot](https://github.com/apps/imgbot)? 🤔

**Comment by takahirom** [comment]
I didn't know that 👀 Let me think it over.

**Comment by takahirom** [comment]
Thanks. I introduced it for trying in this repository.
But as far as this issue is concerned, the images are in `https://user-images.githubusercontent.com/` . So Probably we can't use them for it.

**Comment by Nthily** [comment]
@tatsuyafujisaki  Yes you can, but @takahirom seems to have used imgbot before, 🤔, BTW, I recommend using [this](https://squoosh.app) to optimize images

**Comment by tatsuyafujisaki** [comment]
@Nthily Thank you. I will use Squoosh.
@takahirom Yes, please.:pray:
